### ISSUE TITLE
Trying and failing to pull image (maria-db) on Raspberry Pi 3B+

### ISSUE THREAD

**Issue by Touchdown441** [ct-Open-Source/team-container#20]
I get the following error when I execute "kubectl logs nextcloud-team-nextcloud-db-75f7cd8c8f-qs2ng":

Error from server (BadRequest): container "nextcloud-team-nextcloud-db" in pod "nextcloud-team-nextcloud-db-75f7cd8c8f-qs2ng" is waiting to start: trying and failing to pull image.

The command "kubectl describe pods nextcloud-team-nextcloud-db-75f7cd8c8f-qs2ng" shows:

Events:
  Type     Reason   Age                    From                  Message
  ----     ------   ----                   ----                  -------
Warning  Failed   24m (x246 over 84m)    kubelet, raspberrypi  Error: ImagePullBackOffNormal   Pulling  9m14s (x19 over 84m)   kubelet, raspberrypi  Pulling image "mariadb"
Normal   BackOff  4m16s (x337 over 84m)  kubelet, raspberrypi  Back-off pulling image "mariadb"

I use Raspbian as an Operation System.

**Comment by Touchdown441** [comment]
I use Raspbian as an Operating System.

**Comment by ElvezPelvez** [comment]
Maybe try this image instead:
https://hub.docker.com/r/jsurf/rpi-mariadb/
